docs/contributor/validations.md

```
@@ -319,39 +324,27 @@ This validation runs when:
 severity: warning
 ```
 
-**Use Cases:**
-- Missing optional configuration
-- Best practice recommendations
-- Potential performance issues
-- Informational messages
-
-**Output:**
+Output:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)